src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private static final String BANANA = "banana";
+    private FruitTransaction fruitTransaction;
+    private BalanceOperationHandler balanceOperationHandler = new BalanceOperationHandler();

```

Comment:
Use @BeforeAll to create an instance of BalanceOperationHandler and FruitTransaction